core/src/main/java/net/opentsdb/query/ReadCacheQueryPipelineContext.java

```diff
@@ -526,7 +526,6 @@ void processResults() {
           return;
         }
       }
-      return;
```

Comment:
Conspicuous. Why continue when we previously did not?

Reply:
Cause it turns out we can have a successful query when we have some cache misses :) I had this in there by accident.